docs: improve landing pages (#81)
canonical/charmed-aether-sd-core: docs/explanation/observability.md

Charmed Aether SD-Core fully integrates with Canonical’s Observability Stack (COS) to provide a comprehensive view of the 5G network's operations. Through this integration, operators can effectively monitor network usage, gauge performance metrics, and diagnose potential issues.

> - **Metrics**: All charms within Charmed Aether SD-Core implement the [prometheus_scrape](https://charmhub.io/integrations/prometheus_scrape) charm relation interface, which allows them to expose metrics that are scraped by Prometheus and available in Grafana. This includes metrics about status, subscriber connectivity, throughput, in addition to the default Go runtime metrics.

Reviewer: ```suggestion
- **Metrics**: All charms within Charmed Aether SD-Core implement the [prometheus_scrape](https://charmhub.io/integrations/prometheus_scrape) charm relation interface, which allows them to expose metrics that are scraped by Prometheus. The metrics are available in Grafana including NF status, subscriber connectivity, throughput, in addition to the default Go runtime metrics.
```
